NSS-Day-Cohort-53/tabloid-full-stack-amicably-profound-insects
# added functionality to view tags

#44 View All Tags

Added file Tag.cs to Models directory. 

Added file ITagRepository.cs and TagRepository.cs to Repository directory. 

Added file TagController.cs to Controller Repository.

Added ITagRepository/TagRepository to Startup.cs.

Added file tagManager.js to modules directory.

Added Tag.js and TagList.js to components directory.

Added a tag route path to ApplicationViews.js

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout ap-viewTags
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > When a user logs in and is authenticated, the user should be able to click on the "Tags Management" link in the navbar. The 
       user should be directed to a list of tags in alphabetical order.
